dashboard/test/helpers/pd/workshop_survey_results_helper_test.rb

```diff
@@ -860,4 +860,192 @@ class Pd::WorkshopSurveyResultsHelperTest < ActionView::TestCase
+                'Often',
+                'All the time'
+              ]
+            }
+          }
+        },
+        'Day 2' => {
+          general: {
+            'overallHow' => {
+              text: 'Overall how much did you learn from this course?',
+              answer_type: 'singleSelect',
+              options: [
+                'Almost nothing',
+                'A little bit',
+                'Some',
+                'Quite a bit',
+                'A tremendous amount'
+              ]
+            },
+            'iFeel133' => {
+              text: 'Overall, how do you feel about computer science?',
+              answer_type: 'singleSelect',
+              options: [
+                'Awful',
+                'Not great',
+                'So-so',
+                'Pretty good',
+                'Fantastic'
+              ]
+            }
+          },
+          facilitator: {
+            'howOften56' => {
+              text: 'How often did your facilitator help you?',
+              options: [
+                'Almost never',
+                'Once in a while',
+                'Sometimes',
+                'Often',
+                'All the time'
+              ]
+            }
+          }
+        }
+      },
+      facilitators: @workshop.facilitators.map {|f| [f.id, f.name]}.to_h
+    }
+
+    expected_averages = {
+      'Facilitator Person 1' => {
+        'overallHow' => {
+          this_workshop: 4.64,
+          all_my_workshops: nil
+        },
+        'howOften56' => {
+          this_workshop: 4.75,
+          all_my_workshops: nil
+        },
+        'iFeel133' => {
+          this_workshop: 4.63,
+          all_my_workshops: nil
+        },
+        facilitator_effectiveness: {
+          this_workshop: 1.34,
+          all_my_workshops: 0.0
+        },
+        teacher_engagement: {
+          this_workshop: 0.0,
+          all_my_workshops: 0.0
+        },
+        overall_success: {
+          this_workshop: 0.93,
+          all_my_workshops: 0.0
+        }
+      },
+      'Facilitator Person 2' => {
+        'overallHow' => {
+          this_workshop: 4.64,
+          all_my_workshops: nil
+        },
+        'howOften56' => {
+          this_workshop: 2.07,
+          all_my_workshops: nil
+        },
+        'iFeel133' => {
+          this_workshop: 4.63,
+          all_my_workshops: nil
+        },
+        facilitator_effectiveness: {
+          this_workshop: 0.96,
+          all_my_workshops: 0.0
+        },
+        teacher_engagement: {
+          this_workshop: 0.0,
+          all_my_workshops: 0.0
+        },
+        overall_success: {
+          this_workshop: 0.93,
+          all_my_workshops: 0.0
+        }
+      },
+      questions: {
+        'overallHow' => 'Overall how much did you learn from this course?',
+        'howOften56' => 'How often did your facilitator help you?',
+        'iFeel133' => 'Overall, how do you feel about computer science?'
+      }
+    }
```

Comment:
This is a large fixture for a fairly small test case.  Can/should this live in a factories file, or at least in a helper method outside this test case?  Not even for future reuse, but simply to make reading this case easier?